plugins/rtmp-services/data/services.json

```diff
@@ -1151,17 +1151,10 @@
         },
         {
             "name": "Switchboard Live",
-            "alt_names": [
-                "Switchboard Live (Joicaster)"
-            ],
```

Comment:
Is this changed intentional? This will break old setups that still have the old name set.

Reply:
Hello @gxalpha the `alt_names: "Switchboard Live (Joicaster)"` is no longer valid for Switchboard, so yes it's intentional.
Can you explain what are those breaking changes?  Thanks

Reply:
The settings in OBS store the selected service by its name.
Any user that had selected "Switchboard Live (Joicaster)" before the name was changed to "Switchboard Live" in https://github.com/obsproject/obs-studio/commit/3ec053ef5df885571ba48aac6daf077412ed71b9 would still have that name stored in their OBS settings. OBS uses the `alt_names` to figure out what service that refers to today.
*As far as I know* (and I could be mistaken here), `alt_names` are not displayed anywhere in the UI.

Reply:
Got it, I might need to update my PR to simply remove the word (Joicaster) on the alt_names

Reply:
The `alt_name` should *not* be updated to remove the previous name, as this may potentially break existing user's setups. Only the `name` value is shown in the UI, but alt_name is used to match configurations that are from before the change.